### PR TITLE
Parachains scheduler.rs to FrameV2

### PR DESCRIPTION
**Pull request by ferrell-code**
relates: #2882 

Following the upgrade guidelines here: https://crates.parity.io/frame_support/attr.pallet.html#upgrade-guidelines.

# :warning: Breaking Change :warning: 
From https://crates.parity.io/frame_support/attr.pallet.html#checking-upgrade-guidelines

 >storages now use PalletInfo for module_prefix instead of the one given to decl_storage: Thus any use of this pallet in construct_runtime! should be careful to update name in order not to break storage or to upgrade storage (moreover for instantiable pallet). If pallet is published, make sure to warn about this breaking change.

So users of the `ParaScheduler` pallet must be careful about the name they used in `construct_runtime!.` Hence the runtime-migration label, which might not be needed depending on the configuration of the `ParaScheduler` pallet.

kusama and westend use `ParasScheduler` in `construct_runtime!`,  so I migrated them to `ParaScheduler` 

rococo uses `Scheduler` in `construct_runtime!`, it was migrated to `ParaScheduler`

polkadot does not use `ParaScheduler`.

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: runtime/parachains/src/scheduler.rs [hunk 1]
```diff
@@ -150,66 +148,80 @@ impl CoreAssignment {
...
+	/// Bounded by the number of cores: one for each parachain and parathread multiplexer.
+	///
+	/// The value contained here will not be valid after the end of a block. Runtime APIs should be used to determine scheduled cores/
+	/// for the upcoming block.
+	#[pallet::storage]
+	#[pallet::getter(fn scheduled)]
+	pub(crate) type Scheduled<T> = StorageValue<_, Vec<CoreAssignment>, ValueQuery>; // sorted ascending by CoreIndex.
+
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {}
```
Comment (KiChjang) [inline runtime/parachains/src/scheduler.rs:220]:
```suggestion
```

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -1477,7 +1477,7 @@ construct_runtime! {
 		ParasShared: parachains_shared::{Pallet, Call, Storage} = 52,
 		ParaInclusion: parachains_inclusion::{Pallet, Call, Storage, Event<T>} = 53,
 		ParasInherent: parachains_paras_inherent::{Pallet, Call, Storage, Inherent} = 54,
-		ParasScheduler: parachains_scheduler::{Pallet, Call, Storage} = 55,
+		ParaScheduler: parachains_scheduler::{Pallet, Call, Storage} = 55,
```
Comment (KiChjang) [inline runtime/kusama/src/lib.rs:1480]:
```suggestion
		ParaScheduler: parachains_scheduler::{Pallet, Storage} = 55,
```

File: runtime/rococo/src/lib.rs [hunk 3]
```diff
@@ -226,7 +226,7 @@ construct_runtime! {
 		ParasShared: parachains_shared::{Pallet, Call, Storage},
 		ParaInclusion: parachains_inclusion::{Pallet, Call, Storage, Event<T>},
 		ParasInherent: parachains_paras_inherent::{Pallet, Call, Storage, Inherent},
-		Scheduler: parachains_scheduler::{Pallet, Call, Storage},
+		ParaScheduler: parachains_scheduler::{Pallet, Call, Storage},
```
Comment (KiChjang) [inline runtime/rococo/src/lib.rs:229]:
```suggestion
		ParaScheduler: parachains_scheduler::{Pallet, Storage},
```

File: runtime/westend/src/lib.rs [hunk 4]
```diff
@@ -1067,7 +1067,7 @@ construct_runtime! {
 		ParasShared: parachains_shared::{Pallet, Call, Storage} = 43,
 		ParaInclusion: parachains_inclusion::{Pallet, Call, Storage, Event<T>} = 44,
 		ParasInherent: parachains_paras_inherent::{Pallet, Call, Storage, Inherent} = 45,
-		ParasScheduler: parachains_scheduler::{Pallet, Call, Storage} = 46,
+		ParaScheduler: parachains_scheduler::{Pallet, Call, Storage} = 46,
```
Comment (KiChjang) [inline runtime/westend/src/lib.rs:1070]:
```suggestion
		ParaScheduler: parachains_scheduler::{Pallet, Storage} = 46,
```

**Review by KiChjang — APPROVED** [review]

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**